pyne/dagmc.pyx

```diff
@@ -591,6 +599,80 @@ def get_material_set(**kw):
             mat_ids.add(d['material'])
     return mat_ids
 
+
+def get_cell_material_assignments(hdf5):
```

Comment:
That is much better. Is where I placed the function in the file a reasonable location? I wasn't sure because there seemed to be a few "groups" of functions in the file.